dashboard/package.json

```diff
@@ -100,7 +101,9 @@
     "eslint": "eslint '**/*.{ts,tsx}' --ignore-pattern '**/*.d.ts' --max-warnings=0",
     "eslint-fix": "eslint '**/*.{ts,tsx}' --ignore-pattern '**/*.d.ts' --fix --max-warnings=0",
     "lint": "npm-run-all lint-js lint-css eslint",
-    "ts-compile-check": "tsc -p tsconfig.json --noEmit"
+    "ts-compile-check": "tsc -p tsconfig.json --noEmit",
+    "extract-lang": "formatjs extract 'src/**/*.ts*' --out-file lang/en.json --id-interpolation-pattern '[sha512:contenthash:base64:6]' --format simple",
+    "compile-lang": "formatjs compile-folder lang src/locales/ --ast --format simple"
```

Comment:
are these scripts used?

Reply:
Yes, they are used for extracting the messages using the default values. Then these langs should be compiled in AST for performance purposes. See https://formatjs.io/docs/getting-started/message-extraction/ for more info.